code/modules/clothing/outfits/standard.dm

```diff
@@ -15,6 +15,22 @@
 	shoes = /obj/item/clothing/shoes/combat/swat
 	r_pocket = /obj/item/lighter
 
+/datum/outfit/centcom/admin
+	name = "Department Officer"
+
+	id = /obj/item/card/id/advanced/debug
```

Comment:
This gives them infinite money which is something you should acknowledge in your PR.